src/pfe/file-watcher/server/src/controllers/projectsController.ts

```diff
@@ -936,7 +937,7 @@ export function deleteFolder(dir: string): Promise<void> {
                 Promise.all(files.map((file) => {
                     return deleteFile(dir, file);
                 })).then(() => {
-                    fs.rmdir(dir, (err) => {
+                    rimraf(dir, (err) => {
```

Comment:
If we're using rimraf do we still need the individual file deletions beforehand?

Reply:
The `.nfs<XXXXX>` is generated in the log dir by NFS after we use the individual delete function to delete the log files in the log dir, so it's still necessary to keep the individual delete functio beforehand.

Reply:
Ok, I suppose it can't hurt to keep it